spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'spec_helper'
+
+# class ApplicationTestController < ApplicationController
+#  skip_before_action :require_user!
+# end
+
+RSpec.describe ApplicationController do
+  let(:user) { double User, email: 'test@emailaddress.com' }
+  before(:each) do
+    controller.instance_variable_set(:@current_user, user)
+  end
+
+  it 'get default council name' do
```

Comment:
Sorry for the pickiness, but could the tense of these be changed - adding an -s at the end, e.g. it 'get**s** default council name', it 'receive**s** contact index' in the other one etc.

Reply:
sure no probs